.github/sync-repo-settings.yaml

```diff
@@ -3,7 +3,7 @@
 branchProtectionRules:
 # Identifies the protection rule pattern. Name of the branch to be protected.
 # Defaults to `master`
-- pattern: master
+- pattern: '{master,v3}'
```

Comment:
Huh - I would be surprised if this works - is this in the docs somewhere?  @tmatsuo

Reply:
I assumed it just passes through to the GitHub API.

I can merge and see what happens. :-)